distsql/main_test.go

```diff
@@ -0,0 +1,13 @@
+package distsql
```

Comment:
We need a license header here, see also other files.

Reply:
> We need a license header here, see also other files.

thanks,fixed and update